src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
+const resolved = document.createElement('div');
 
-// write your code here
+resolved.classList.add('message');
+resolved.textContent = 'Promise was resolved!';
+
+const rejected = document.createElement('div');
+
+rejected.classList.add('message');
+rejected.classList.add('message-error');
```

Comment:
The class name should be `error-message` instead of `message-error` to align with the task requirements.